apps/web/pages/booking/[uid].tsx

```diff
@@ -197,12 +199,10 @@ export default function Success(props: SuccessProps) {
     formerTime,
     email,
     seatReferenceUid,
+    tz: timeZoneQuery,
   } = querySchema.parse(router.query);
 
-  const tz =
-    (isSuccessBookingPage
-      ? props.bookingInfo.attendees.find((attendee) => attendee.email === email)?.timeZone
```

Comment:
Can't we leave it like that to show the attendee's timezone when `isSuccessBookingPage` is true, instead of passing it through the query?

If `isSuccessBookingPage` is false then we show the tz of the logged in user (as you did it)

Reply:
added tz to the query for a edges case , if a authenticated user booked in a different timezone rather than his default one